lib/datadome/configuration.rb

```diff
@@ -9,13 +9,13 @@ def initialize
       @api_server = "api.datadome.co"
       @exclude_matchers = []
       @include_matchers = []
-      @monitor_mode = false
```

Comment:
Can we keep both maybe?

Reply:
With below logic for instance ?

Intercept if 
```ruby
(bot request) and (monitor mode disabled OR intercept match ?)
```

Reply:
yes

Reply:
done.